### PR TITLE
[RFE]Updating RHCloud tests to SCA only

### PR DESCRIPTION
**Pull request by ColeHiggins2**
Updating rhcloud related tests to run to be SCA only.

### CODE REVIEW

File: pytest_fixtures/component/rh_cloud.py [hunk 1]
```diff
@@ -4,10 +4,10 @@
 
 
 @pytest.fixture(scope='module')
-def rhcloud_manifest_org(module_target_sat, module_extra_rhel_entitlement_manifest):
```
Comment (jameerpathan111) [inline pytest_fixtures/component/rh_cloud.py:7]: @ColeHiggins2 should we cleanup the code and remove `module_extra_rhel_entitlement_manifest` fixture as it was only used by rhc tests? wdyt?

**Review by jameerpathan111 — APPROVED** [review]

File: pytest_fixtures/component/rh_cloud.py (hunk 1 above)
Comment (jameerpathan111) [reply pytest_fixtures/component/rh_cloud.py:7]: also `module_extra_rhel_entitlement_manifest_org`

**Comment by synkd** [comment]
> There doesn't seem to be many tests that use entitlements so this should be all the work.
> 
> Additionally, Should we cherry-pick this into 6.15? I am open to suggestions

I would recommend running the affected tests using SCA fixtures against 6.15, 6.14, and 6.13. If they're not affected by switching to SCA, I don't see a reason not to cherrypick all the way back to 6.13.

File: pytest_fixtures/component/rh_cloud.py [hunk 2]
```diff
@@ -4,10 +4,10 @@
...
+def rhcloud_manifest_org(module_target_sat, module_extra_rhel_sca_manifest):
     """A module level fixture to get organization with manifest."""
     org = module_target_sat.api.Organization().create()
-    module_target_sat.upload_manifest(org.id, module_extra_rhel_entitlement_manifest.content)
+    module_target_sat.upload_manifest(org.id, module_extra_rhel_sca_manifest.content)
```
Comment (synkd) [inline pytest_fixtures/component/rh_cloud.py:10]: `module_extra_rhel_sca_manifest` uses the `extra_rhel_entitlement` manifest category [1], so this change will not reduce the subscription footprint of the rhcloud tests. Please use `module_sca_manifest` with this fixture.

[1] https://github.com/SatelliteQE/robottelo/blob/21b5935ddfeb7f3a8cf36d1acc0cf0a064c97036/pytest_fixtures/component/taxonomy.py#L204

File: pytest_fixtures/component/rh_cloud.py (hunk 1 above)
Comment (synkd) [reply pytest_fixtures/component/rh_cloud.py:7]: Yes, we should remove all of the `extra_rhel` fixtures for stream/6.16. There is no benefit to using more than 1 of each subscription on an SCA manifest. If the affected tests pass with SCA on earlier Satellite versions, we can cherrypick that change back as well.

**Review by synkd — CHANGES_REQUESTED** [review]
Requesting a couple of changes. Also, I see about a dozen tests will be affected by this PR. Most of these tests are failing consistently in CI, so there may not be much value in testing them with PRT, but I would like to ensure that the passing tests continue passing with these changes. Please run a PRT job with these tests:
- tests/foreman/cli/test_http_proxy.py::test_insights_client_registration_with_http_proxy
- tests/foreman/ui/test_rhcloud_insights.py::test_insights_registration_with_capsule
- tests/foreman/ui/test_rhc.py::test_positive_configure_cloud_connector

**Review by vijaysawant — APPROVED** [review]

File: pytest_fixtures/component/rh_cloud.py (hunk 1 above)
Comment (ColeHiggins2) [reply pytest_fixtures/component/rh_cloud.py:7]: I have another jira ticket that cleans up all fixtures and code around sca. so I will do it then

**Comment by ColeHiggins2** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_rhcloud_insights.py tests/foreman/ui/test_rhc.py tests/foreman/cli/test_http_proxy.py -k 'test_insights_client_registration_with_http_proxy or test_insights_registration_with_capsule or test_positive_configure_cloud_connector'

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7881
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_rhcloud_insights.py tests/foreman/ui/test_rhc.py tests/foreman/cli/test_http_proxy.py -k test_insights_client_registration_with_http_proxy or test_insights_registration_with_capsule or test_positive_configure_cloud_connector --external-logging
Test Result : ======== 13 passed, 15 deselected, 1572 warnings in 10469.36s (2:54:29) ========
```

**Review by synkd — APPROVED** [review]

**Review by vsedmik — APPROVED** [review]

**Review by pondrejk — APPROVED** [review]

**Pull request merged**